nvchecker/archlinux-proaudio.toml

```diff
@@ -34,6 +34,12 @@ source = "github"
 github = "sdangelo/asid"
 use_max_tag = true
 
+# note: we use the stable website downloads for building, but gitlab for checking
+[audiowmark]
+source = "gitlab"
+gitlab = "stefan/audiowmark"
+use_max_tag = true
```

Comment:
this needs `host = "code.uplex.de"` and probably an `include_regex` because there is one non-version tag in that repo